app/controllers/recipes_controller.rb

```diff
@@ -0,0 +1,55 @@
+class RecipesController < ApplicationController
+  before_action :set_recipe, only: %i[show edit update destroy]
+
+  def index
+    if user_signed_in?
+      @recipes = Recipe.where(user_id: current_user.id)
+    else
+      redirect_to new_user_session_path
+    end
+  end
+
+  def show; end
+
+  def new
+    @recipe = Recipe.new
+  end
+
+  def create
+    @recipe = Recipe.new(recipe_params)
+    if @recipe.save
+      redirect_to recipes_path, notice: 'Recipe was successfully created.'
+    else
+      render :new, status: :unprocessable_entity
+    end
+  end
+
+  def update
+    if @recipe.update(params.require(:recipe).permit(:public))
+      flash.now[:notice] = 'Recipe was successfully updated.'
+    else
+      flash.now[:error] = 'Recipe was not updated.'
+    end
+  end
+
+  def destroy
+    @recipe.destroy
+    redirect_to recipes_url, notice: 'Recipe was successfully destroyed.'
+  end
```

Comment:
- [ ]  I also get the same error as above here. Please kindly help fix this. This can be resolved by including `skip_before_action :verify_authenticity_token` in the controllers

![image](https://user-images.githubusercontent.com/43995830/175456004-68a71538-5d79-449b-9f3b-af5c7bbe75a8.png)